src/main/svg2png.ts

```diff
@@ -244,8 +267,10 @@ class Svg2png {
       await this.throwIfCancelled(page);
 
       this.status = Status.PENDING;
+      this.tic();
       this.log(`navigating to page`, { url: this.source });
       await this.navigateToSource(page);
+      this.log(`    ${this.toc()}ms`);
```

Comment:
any reason to not use the console.time (even if it is in tic/toc methods

Reply:
Not really, I guess I didn't want to deal with the string ids. On second thought, maybe I'll just switch to console.time in the toc method.